_pages/team.html

```diff
@@ -98,6 +98,18 @@ <h5>Carolyn Baek</h5>
             <h5>Ethan Sisbarro</h5>
             <p>Ethan works on game strategy.</p>
         </div>
+        <div>
+            <img src="/assets/img/team/nikita-solovyov.png">
+            <h5>Nikita Solovyov</h5>
+            <!---TODO: add detials.--->
+            <p></p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Anuragi Thapliya</h5>
+            <!---TODO: add detials.--->
```

Comment:
We already have Anuragi, Carolyn, and maybe a couple others. Check if there are any others who are already on the website: http://1418.team/team